post/release-0-2.qmd

```diff
@@ -1,8 +1,15 @@
 ---
 title: "GeoArrow 0.2 Release"
+author:
+  - name: Dewey Dunnington
+    url: https://dewey.dunnington.ca/
+  - name: Kyle Barron
+    url: https://kylebarron.dev/
+  - name: Joris Van den Bossche
+    url: https://jorisvandenbossche.github.io/
```

Comment:
I think it makes sense to put our names on the post. Happy to switch to email instead of URL but I think useful to be able to hyperlink something